cart/src/main/resources/application.properties

```diff
@@ -2,16 +2,18 @@ server.port=8084
 server.servlet.context-path=/cart
 
 spring.application.name=cart
-spring.sleuth.otel.config.trace-id-ratio-based=1.0
-spring.sleuth.otel.exporter.otlp.endpoint=http://jaeger:30017
+management.tracing.sampling.probability=1.0
+management.endpoints.web.exposure.include=prometheus
+
+logging.pattern.level=%5p [${spring.application.name:},%X{traceId:-},%X{spanId:-}]
 
 spring.security.oauth2.resourceserver.jwt.issuer-uri=http://identity/auth/realms/Yas
 
 yas.services.media=http://api.yas.local/media
 yas.services.product=http://api.yas.local/product
 
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:30007/cart
+spring.datasource.url=jdbc:postgresql://localhost:5432/cart
```

Comment:
fix this port for the docker

Reply:
When we build docker images, our database will run in port 30007, not 5432.

Reply:
Please help to read README.MD again. In local env for test purpose Postgres also listen on port 30007. I think you should use docker-compose b/c Dockerfile no longer updated.

Reply:
Oh, sorry about my mistake.